recipe/silverstripe.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+require_once __DIR__ . '/common.php';
+
+/**
+ * Silverstripe configuration
+ */
+
+// Silverstripe shared dirs
+set('shared_dirs', [
+    'assets'
+]);
+
+// Silverstripe writable dirs
+set('writable_dirs', ['assets']);
+
+/**
+ * Helper tasks
+ */
+task('silverstripe:build', function () {
+    return run('{{bin/php}} {{release_path}}/framework/cli-script.php /dev/build');
```

Comment:
Sure, 

In Silverstripe you create database tables in code these are [DataObject](http://api.silverstripe.org/3.3/class-DataObject.html) classes.

So the class
```php
class MyObject extends DataObject
{
    private static $db            = array(
        "Name"             => "VarChar(50)"
    );
}
```

Has the database tabe (mysql)
```sql
CREATE TABLE `MyObject ` (
  `Name` varchar(50) DEFAULT NULL
) 
```

But just adding a new class will not create or update the database schema. 

Running `/dev/build` make Silverstripe compare all [DataObjects](http://api.silverstripe.org/3.3/class-DataObject.html) and updates/creates the underlining database tables as required.